concrete/src/Page/Theme/BedrockThemeTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Concrete\Core\Page\Theme;
+
+trait BedrockThemeTrait
```

Comment:
This trait could use a docblock at the top and for all members making it clear why you'd use this and what these methods do